pubspec.lock

```diff
@@ -1521,5 +1529,5 @@ packages:
     source: hosted
     version: "2.1.1"
 sdks:
-  dart: ">=3.0.5 <4.0.0"
+  dart: ">=3.2.0-194.0.dev <4.0.0"
```

Comment:
This cannot be right I think.
When I do fvm flutter pub get it sets this to dart: ">=3.0.5 <3.1.0" which also aligns with what you wrote in the pubspec.yaml file.
Did you update this file too after changing the pubspec.yaml?